webpack/envs/clientCommonConfig.js

```diff
@@ -202,7 +202,7 @@ export const clientCommonConfig = {
           name: "common",
           chunks: "all",
           minSize: 0,
-          minChunks: 1,
+          minChunks: 2,
```

Comment:
Rather than this broad change can we use Webpack's `Ignore` plugin instead? and then just import openseadragon directly into a file that the artwork page includes?

Reply:
`minChunks: 1` does seem overly aggressive for commons though?

Reply:
I wouldn't say this is a broad change, I reduced `minChunks` from `10` to `1` in the previous PR knowing that it would likely need fine tuning. So far I like the results of 2, I don't want all of `node_modules` to exist in the common chunk at the moment. That approach may make sense in the future when we have smaller, better defined artifact. Right now it means that every 10KiB package used used one on the site will be loaded for every page impression.

Reply:
I tend to lean towards `WebpackIgnore` when excluding source that never should have been included in the first place. In the future it may be worth exploring using Webpack magic comments to lazy load these types of packages. 

```
/* webpackChunkName: "my-chunk-name" */
```

Magic comments have their advantages and disadvantages. I personally feel like `minChunks: 2` strikes a good balance here.